app/controllers/wpcc/your_results_controller.rb

```diff
@@ -47,19 +43,22 @@ def message_presenter
     end
     helper_method :message_presenter
 
-    def period_filter_presenter
-      Wpcc::PeriodFilterPresenter.new(period_filter, view_context: view_context)
+    def period_percents_mapper_presenter
```

Comment:
FYI, I've deleted this all instances of its usage in this PR https://github.com/moneyadviceservice/wpcc/pull/222